static/src/javascripts/projects/common/modules/navigation/newSticky.js

```diff
@@ -107,48 +113,36 @@ define([
     };
 
     var render = function (els, state) {
-        // Reset so we have a clean slate
-        els.$header.css({
-            'transition': '',
-            'margin-top': ''
-        });
-        els.$adBanner.css({
-            'position': '',
-            'top': '',
-            'max-height': '',
-            'overflow': '',
-            'transition': ''
-        });
-
-        // Set
         // TODO: move into stylesheets when productionised
         els.$document.addClass('new-sticky-ad');
-        els.$header.css({ 'margin-top': state.adHeight });
-        els.$adBanner.css({ 'max-height': state.adHeight });
+
+        var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
+        var transitionDuration = '1s';
+
+        els.$header.css({
+            'transition': state.shouldTransition
```

Comment:
Would this not set the transition to an invalid CSS value?
Perhaps just not setting it if `!shouldTransion`.